java/arcs/core/host/ParticleIdentifier.kt

```diff
@@ -1,5 +1,6 @@
 package arcs.core.host
 
+import arcs.core.host.api.Particle
```

Comment:
Needed?

Reply:
Yes, I refactored itinto a subpackage, it used to be in the host package and so didn't need an import.